Source/WebCore/Sources.txt

```diff
@@ -2551,19 +2551,25 @@ rendering/style/ShapeValue.cpp
 rendering/style/StyleBackgroundData.cpp
 rendering/style/StyleBoxData.cpp
 rendering/style/StyleCachedImage.cpp
+rendering/style/StyleCanvasImage.cpp
```

Comment:
I'm starting to think we should call all computed style related things "computed", like ComputedCanvasImage (including RenderStyle -> ComputedStyle). This would clarify what sort of style we are talking about using a spec term.

The patch is fine as-is.